openmc/capi/core.py

```diff
@@ -269,19 +269,21 @@ def source_bank():
     return as_array(ptr, (n.value,)).view(bank_dtype)
 
 
-def statepoint_write(filename=None):
+def statepoint_write(filename=None, write_source=False):
```

Comment:
Is there a reason that we shouldn't just default to writing the source? (since that's the default with normally-generated statepoints?)